install

```diff
@@ -226,24 +225,24 @@ end
 
 wait_for_user if STDIN.tty? && !ENV["TRAVIS"]
 
-if File.directory? HOMEBREW_PREFIX
-  sudo "/bin/chmod", "g+rwx", *group_chmods unless group_chmods.empty?
-  sudo "/bin/chmod", "u+rwx", *user_chmods unless user_chmods.empty?
-  sudo "/usr/sbin/chown", ENV["USER"], *chowns unless chowns.empty?
-  sudo "/usr/bin/chgrp", "admin", *chgrps unless chgrps.empty?
```

Comment:
If HOMEBREW_PREFIX didn't exist none of those directories have the wrong permissions at this stage so, like before, there's only any need to fix up the ZSH directory permissions after they were created or if they already existed (neither applies here).